src/validation/tests/assignment_validation_tests.rs

```diff
@@ -554,10 +554,13 @@ fn array_assignment_validation() {
         v_arr_real_3 : ARRAY[0..3] OF REAL;
     
         v_arr_string_3 : ARRAY[0..3] OF STRING;
+        v_arr_sized_string : ARRAY[0..3] OF STRING[256];
+        v_arr_sized_string1 : ARRAY[0..3] OF STRING[256];
     
         v_arr_char_3 : ARRAY[0..3] OF CHAR;
     END_VAR
     // ARRAY
+    v_arr_sized_string := v_arr_sized_string1; // valid
```

Comment:
could you a test for the invalid case where the arrays have the same type but different sizes?
So i would say
```iecst
 v_arr_sized_string2 : ARRAY[0..3] OF STRING[1256];
 v_arr_sized_string3 : ARRAY[0..4] OF STRING[256];
```